api/model/storage/postgres/request.go

```diff
@@ -67,8 +67,8 @@ func (s *Storage) PersistRequestFilters(requestID string, filters *api.FilterPar
 			if err != nil {
 				return errors.Wrap(err, "failed to persist numerical filter")
 			}
-		case model.BivariateFilter:
-			_, err := s.client.Exec(sql, requestID, filter.Key, model.BivariateFilter, filter.Mode, 0, 0, filter.Bounds.MinX, filter.Bounds.MaxX, filter.Bounds.MinY, filter.Bounds.MaxY, "", "")
+		case model.BivariateFilter, model.GeoBoundsFilter:
```

Comment:
Can BivariateFilter be completely removed from the codebase or is it still used somewhere?

Reply:
I dont think it is used somewhere, so if that means it should be removed then so be it. Can you think of a use case where we could want that kind of filtering?

Reply:
I guess if we wanted to do general 2D plots instead of geo?